--- a/lib/variform/validation/method_allowlist.rb
+++ b/lib/variform/validation/method_allowlist.rb
@@ -10,7 +10,6 @@
         annotate
         antialias
         append
-        apply
         auto_gamma
         auto_level
         auto_orient
@@ -64,7 +63,6 @@
         level
         limit
         liquid_rescale
-        loader
         magnify
         median
         minify
@@ -97,7 +95,6 @@
         rotate
         sample
         sampling_factor
-        saver
         scale
         screen
         sepia_tone
